tests/queries/0_stateless/03594_text_index_size_in_data_skipping_indices.sql

```diff
@@ -0,0 +1,37 @@
+SET allow_experimental_full_text_index = 1;
```

Comment:
(a minor thing:
- text index tests start with prefix 02346_
- if they test a bug, we should name them after the bug number --> 02346_text_index_bug62681.sql

)